README.rst

```diff
@@ -426,6 +426,28 @@ FAQ
     # downloading a specific rpms built on master in job 888
     export SCT_UPDATE_DB_PACKAGES=s3://downloads.scylladb.com/rpm/unstable/centos/master/888/scylla/7/x86_64/
 
+**Q:** How to use SCT_SCYLLA_MGMT_PKG and what does it do?
+
+**A:** SCT has the ability to run a job (manager jobs) using your own scylla-manager package files. It will allow you to run one of the manager jobs using your self build package files, for example ::
+
+    # uploading to s3
+    aws s3 cp --recursive rpms s3://scylla-qa-public/`whoami`/
+
+    # download from s3 path
+    export SCT_SCYLLA_MGMT_PKG=s3://scylla-qa-public/`whoami`/rpms
+
+    # uploading to google storage
+    gsutil cp rpms/* gs://scratch.scylladb.com/`whoami`/rpms/
+
+    # download for google storage
+    export SCT_SCYLLA_MGMT_PKG=gs://scratch.scylladb.com/`whoami`/rpms
```

Comment:
Doesn't we need a separate config param for the agent ? and on for the manager ?
since with this one url, we need to put both .deb and .rpms  in the same s3 directory, we at least should document this option, if not adding a config param

Reply:
nope, i consider all packages built for manager, including server, client and agent will be place together in the same folder, as we download it to the builder, prior to send them to the right place, so i prefer to deal less with downloading.
and because the user providing the packages can put in this folder any file compiled by him/her, i won't consider another param for agent separated from the client and server ones.

Reply:
I was referring to the mix of debian / centos for example.
but good enough if you think it would be clear enough for whom would need it.